pages/docs/reference/this-expressions.md

```diff
@@ -9,8 +9,8 @@ title: "This expressions"
 
 To denote the current _receiver_, we use *this*{: .keyword } expressions:
 
-* In a member of a [class](classes.html#inheritance), *this*{: .keyword } refers to the current object of that class
-* In an [extension function](extensions.html) or a [function literal with receiver](lambdas.html#function-literals-with-receiver),
+* In a member of a [class](classes.html#inheritance), *this*{: .keyword } refers to the current object of that class;
+* In an [extension function](extensions.html) or a [function literal with receiver](lambdas.html#function-literals-with-receiver);
```

Comment:
This semicolon seems to be put by accident: the next lines continues the same sentence. Also, the [comment about list items with sentences](https://github.com/JetBrains/kotlin-web-site/pull/885/files#r139399030) is relevant here as well,